index.js

```diff
@@ -0,0 +1,192 @@
+const Manager = require("./lib/Manager");
```

Comment:
great job importing all of the classes and packages external to this file at the top - fantastic for readability :-)